howso/utilities/feature_attributes/relational.py

```diff
@@ -804,6 +804,18 @@ def _infer_feature_bounds(self,  # noqa: C901
             column_type, _ = self._parse_column_type(str(column.type))
             format_dt = None
 
+            # Compute loose time-only feature bounds
+            if feature_attributes[feature_name]['data_type'] == 'formatted_time':
+                if (
```

Comment:
The pandas version handles tight bounds as well, should we do the same here?

Reply:
I was thinking that the function already handled the tight bounds (since time data is well-supported by sqlalchemy, as opposed to Pandas), but it wouldn't actually give the min/max in seconds. Fixed now.